content/updates/terraform-controller-overview.md

```diff
@@ -0,0 +1,210 @@
+---
+visible: true
+draft: false
+title: "Mastering GitOps with Terraform Controller"
+author: Veronika Gnilitska
+slug: terraform-controller-overview
+date: TBD
+description: The post explores how Weave's Terraform Controller can enable GitOps for infrastructure management.
+image: /img/updates/terraform-controller-overview/tf_controller_0.png
+callout: "<p>👋 <b>Interested in platform engineering for your organization</b>, but not sure where to start? <a href='/contact'>Get in touch,</a> we're an expert team of platform engineers who deliver high-quality cloud platforms for startups and SMBs looking to scale. We enable your application engineers to focus on your product and in turn generate more value for your business.</p><a href='/contact' class='button'>Get In Touch &rsaquo;</a>"
+---
+<h2>Table of Contents</h2>
```

Comment:
Doesn't seem like this TOC is rendering correctly in the preview... we may need to cut it unless you know how it should be working?

Reply:
I was missing an empty row there, this is fixed. However, these extra spaces in the least important nested headings drive me nuts. I've noticed the similar behavior in the text, so I guess it's something we can fix on netlify side (?).
<img width="678" alt="Screenshot 2024-01-30 at 16 04 20" src="https://github.com/masterpointio/masterpoint.io/assets/30597968/6a50f764-5143-4564-978f-a0a933b6b7d1">

I love TOCs, they really help to evaluate if I'm interested in the content. But I have doubts if we should release it like that.